modules/sitemaps/sitemap-stylist.php

```diff
@@ -0,0 +1,660 @@
+		$header_title = esc_html( ent2ncr( __( 'Title', 'jetpack' ) ) );
+		$header_pubdate = esc_html( ent2ncr( __( 'Publication Date', 'jetpack' ) ) );
+
+		$generated_by = wp_kses(
+			sprintf(
+				ent2ncr(
+					__(
+						'<em>Generated</em> by <a href="%s" target="_blank">Jetpack for WordPress</a>',
+						'jetpack'
+					)
+				),
+				'https://jetpack.com'
+			),
+			array( 'a' => array( 'href' => true, 'title' => true ) )
+		);
+
+		$css = self::sitemap_xsl_css();
+
+		return <<<XSL
+<?xml version='1.0' encoding='UTF-8'?>
+<xsl:stylesheet version='2.0'
+  xmlns:html='http://www.w3.org/TR/REC-html40'
+  xmlns:sitemap='http://www.sitemaps.org/schemas/sitemap/0.9'
+  xmlns:news='http://www.google.com/schemas/sitemap-news/0.9'
+  xmlns:xsl='http://www.w3.org/1999/XSL/Transform'>
+<xsl:output method='html' version='1.0' encoding='UTF-8' indent='yes'/>
+<xsl:template match="/">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <title>$title</title>
+  <meta http-equiv='Content-Type' content='text/html; charset=utf-8'/>
+  <style type='text/css'>
+$css
+  </style>
+</head>
+<body>
+  <div id='description'>
+    <h1>$title</h1>
+    <p>$description</p>
+    <p>$more_info</p>
+  </div>
+  <div id='content'>
+    <!-- <xsl:value-of select="count(sitemap:urlset/sitemap:url)"/> -->
+    <table>
+      <tr>
+        <th>#</th>
+        <th>$header_url</th>
+        <th>$header_title</th>
+        <th>$header_pubdate</th>
+      </tr>
+      <xsl:for-each select="sitemap:urlset/sitemap:url">
+        <tr>
+          <xsl:choose>
+            <xsl:when test='position() mod 2 != 1'>
+              <xsl:attribute name="class">odd</xsl:attribute>
+            </xsl:when>
+          </xsl:choose>
+          <td>
+            <xsl:value-of select = "position()" />
+          </td>
+          <xsl:variable name='pageURL'>
+            <xsl:value-of select='sitemap:loc'/>
+          </xsl:variable>
+          <td>
+            <a href='{\$pageURL}'>
+              <xsl:value-of select='sitemap:loc'/>
+            </a>
+          </td>
+          <td>
+            <a href='{\$pageURL}'>
+              <xsl:value-of select='news:news/news:title'/>
+            </a>
+          </td>
+          <td>
+            <xsl:value-of select='news:news/news:publication_date'/>
+          </td>
+        </tr>
+      </xsl:for-each>
+    </table>
+  </div>
+  <div id='footer'>
+    <p>$generated_by</p>
+  </div>
+</body>
+</html>
+</xsl:template>
+</xsl:stylesheet>\n
+XSL;
+	}
+
+	/**
+	 * The CSS to be included in sitemap xsl stylesheets;
+	 * factored out for uniformity.
+	 *
+	 * @access public
+	 * @since 4.5.0
+	 */
+	public static function sitemap_xsl_css() {
+		return <<<CSS
+    body {
+      font: 14px 'Open Sans', Helvetica, Arial, sans-serif;
+      margin: 0;
+    }
+
+    a {
+      color: #3498db;
+      text-decoration: none;
+    }
+
+    h1 {
+      margin: 0;
+    }
+
+    #description {
```

Comment:
The selectors `#description, #content, .odd, #footer, img.thumbnail` are never used.